docset/winserver2022-ps/deduplication/Stop-DedupJob.md

```diff
@@ -175,14 +191,14 @@ Accept wildcard characters: False
 ```
 
 ### -Volume
-Specifies one or more file system volumes on which to cancel one or more named data deduplication jobs.
-Enter one or more volume IDs, drive letters, or volume GUID paths.
-For drive letters, use the format D:.
-For volume GUID paths, use the format \\\\?\Volume{{GUID}}\.
-Separate multiple volumes with a comma.
+
+Specifies one or more file system volumes on which to cancel one or more named data deduplication
+jobs. Enter one or more volume IDs, drive letters, or volume GUID paths. For drive letters, use the
+format `D:`. For volume GUID paths, use the format `\\\\?\Volume{{GUID}}\`. Separate multiple
+volumes with a comma.
```

Comment:
The initial doubling of backslashes was to escape them in the raw Markdown. Now that we're wrapping these in code, the extra backslashes should be omitted.

```suggestion
Specifies one or more file system volumes on which to cancel one or more named data deduplication
jobs. Enter one or more volume IDs, drive letters, or volume GUID paths. For drive letters, use the
format `D:`. For volume GUID paths, use the format `\\?\Volume{{GUID}}\`. Separate multiple
volumes with a comma.
```